resources/metadata/kubernetesDeploy.yaml

```diff
@@ -460,6 +481,41 @@ spec:
         possibleValues:
           - apply
           - replace
+      - name: setupScript
+        type: string
+        description: HTTP location of setup script
+        longDescription: |
+          For helm-based deploymens only!
+          HTTP location of setup script.
+          The script will be downloaded from a GitHub location using the `githubToken` and executed before the installation of the helm package.
+        scope:
+          - PARAMETERS
```

Comment:
should we add a general scope as well, so that the same script can be used in all stage acceptance / performance by adding it once in the config ?

Reply:
I am not sure. The scripts will likely be relevant mostly for testing. Let's maybe wait until we see clear needs, what do you think?